Coding/fw/lib/Lawicel/Lawicel.h

```diff
@@ -141,9 +141,10 @@ class Lawicel
     bool getSerialNumberCmd(const String &lawicelCMD);   /**< Sends Serial Number of Hardware */
     bool toggleTimeStampCmd(const String &lawicelCMD);   /**< Toggles Timestamp (and saves setting on EEPROM) */
     bool toggleAutoStartCmd(const String &lawicelCMD);   /**< Auto Startup feature (from power on) */
+    bool getCurrentParams(const String &lawicelCMD);     /**< Returns the current configuration of the CAN Bus */
 
-    bool autopoll();      /**< Frame Polling without any extra tags */
-    uint32_t getTimestamp(); /**< Returns Timestamp */
+    bool autopoll();       /**< Frame Polling without any extra tags */
+    String getTimestamp(); /**< Returns Timestamp */
```

Comment:
i like the old version more... 
Why is it a string now ?
if it needs to be string, call it formatTimeStamp() instead of getTimestamp()